### ISSUE TITLE
[0.7.4.0 edge] Typo in Mods\AutoGen\WorldObject\TailoringTable.cs

### ISSUE THREAD

**Issue by Thineboot** [StrangeLoopGames/EcoIssues#7649]
**�** in Description  
```
Line 64:
    public partial class TailoringTableItem : WorldObjectItem<TailoringTableObject>
    {
        public override string FriendlyName { get { return "Tailoring Table"; } } 
        public override string Description  { get { return  "Spearhead the fashion movement with the finest clothes and� hair!"; } }

        static TailoringTableItem()
        {       
        }
    }
```

is ok in [0.7.4.0 staging]

**Comment by Thineboot** [comment]
Fixed in 0.7.4.0

**Comment by craig-jong** [comment]
see response to https://github.com/StrangeLoopGames/EcoIssues/issues/7631
